.buildkite/dagster-buildkite/dagster_buildkite/cli.py

```diff
@@ -19,20 +21,26 @@ def dagster():
     if not dagit_only:
         all_steps += dagster_steps()
 
-        all_steps += [wait_step()]
-
-        if DO_COVERAGE:
-            all_steps += [coverage_step()]
-
-        # Trigger builds of the internal pipeline for builds on master
+        # Trigger builds of the internal pipeline.
```

Comment:
It'd be nice to land this buildkite stuff as a separate commit just so it's more visible in the commit history maybe?

Reply:
🤔 I don't think this is in the PR...

Reply:
Oh, it's probably because the base branch changed.